src/nvim/README.md

```diff
@@ -23,14 +23,20 @@ Logs
 
 Low-level log messages sink to `$NVIM_LOG_FILE`.
 
-Use `LOG_CALLSTACK()` (Linux only) to log the current stacktrace. To log to an
-alternate file (e.g. stderr) use `LOG_CALLSTACK_TO_FILE(FILE*)`.
-
 UI events are logged at DEBUG level (`DEBUG_LOG_LEVEL`).
 
     rm -rf build/
     make CMAKE_EXTRA_FLAGS="-DMIN_LOG_LEVEL=0"
 
+Use `LOG_CALLSTACK()` (Linux only) to log the current stacktrace. To log to an
+alternate file (e.g. stderr) use `LOG_CALLSTACK_TO_FILE(FILE*)`.
+This 2 functions use `addr2line` internally so they require nvim to be compiled
+with
+`-no-pie` ([ref](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=860394#15)):
+
+    rm -rf build/
+    make CMAKE_EXTRA_FLAGS="-DMIN_LOG_LEVEL=0 -DCMAKE_C_FLAGS=-no-pie"
```

Comment:
hmm. perhaps we should enforce this in CMakeLists.txt ...

Reply:
Do you want to make debug build `-no-pie` by default just to make `addr2line` work? Seem like little overkill, `LOG_CALLSTACK` is not used in source code, it hast to be manually added if one wants to debug/hack something. Is there any other benefit of `-no-pie` in debug build?

Reply:
ideally we could somehow detect the lack of `-no-pie` in `LOG_CALLSTACK` and then show a build-time error there. Doubt that's possible I guess. But generally we want to avoid burdening the docs with lots of special case notes, and instead use mechanisms to warn users just-in-time.